data/binding/bindlists.go

```diff
@@ -1,9 +1,12 @@
 // auto-generated
 // **** THIS FILE IS AUTO-GENERATED, PLEASE DO NOT EDIT IT **** //
```

Comment:
I think you missed this line. You need to to update the file that generates this file.

Reply:
I did notice it, but at that moment hadn't figured out how to generate those. Now I know, and notice, that my current skills are not good enough with this gen-stuff... Now every External list has `DeepEqual`, which is silly, say for boolean arrays. Is there a way I could control the generation so that for ExternalUntyped it would use DeepEquals?

And another thing I noticed: my PR is against master instead of develop as instructed. Should I delete this PR, and start over.

Reply:
I think I wrote a another comment on the PR while you were sending yours here. It should be possible to fix up this PR but you can also create a new if you feel like that is easier.

Hmm, I'll have to look at that. I don't know how easy that will be.

Reply:
Replying here to keep the context: after checking out how the generation works, I found out, that it was pretty easy: the `bindValues` struct in `gen.go` contains a field, `Comparator`, and if that field is empty equality is done with `==`, and if it is not empty, the given string is used as a function.